client/src/components/FooterComponent/FooterComponent.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+const FooterComponent = ({ message }) => {
```

Comment:
Let's name it Footer. It's redundant as it is already placed inside components directory.

Reply:
Done